frontend/src/modules/alarms/AlarmsPage.tsx

```diff
@@ -396,7 +440,61 @@ export const AlarmsPage = (): JSX.Element => {
                         <Typography variant="h4">Confirm New Changes?</Typography>
                       </Grid>
                     </Grid>
-                    <Grid item alignItems="center" className={classes.marginContent} />
+                    <Grid item className={classes.marginContent}>
+                      {alarmConfig.map((param) => {
+                        if (
+                          alarmLimits[param.stateKey].lower !==
```

Comment:
Shouldn't these be comparing `alarmLimitsRequestDraft` against `alarmLimitsRequest`, not `alarmLimits`? Maybe that could explain why we're seeing buggy behavior in test cases 7 and 8.

Reply:
I'm resolving for now as I'm using a selector now, if there is a better way to do that I will fix it there.